iris-mpc-gpu/src/server/actor.rs

```diff
@@ -406,6 +407,9 @@ impl ServerActor {
             dev.synchronize().unwrap();
         }
 
+        let anonymized_bucket_statistics =
+            BucketStatistics::new(match_distances_buffer_size, n_buckets, party_id);
```

Comment:
the `start_timestamp` is then always the start time of the server, is this intended?